RELEASE.md

```diff
@@ -38,12 +43,9 @@ automated and require no human action.
   candidate
   4. Depending on the cherry-picked changes, it may be necessary to perform some
   or all of the manual tests again.
-7. Once happy with the release-candidate, perform the release using the release
-  script located at `scripts/release.sh` (or perform the steps in that script
-  manually):
-  - `./scripts/release.sh v0.8.26`
-8. NOACTION: The HEAD of the `release` branch will be tagged with `v0.8.26`,
-  and a final draft release will be created on Github.
+7. Once happy with the release-candidate (i.e., the checks in the auto-created
+  checklist are completed) tag the tip of the release-candidate branch:
+  - `git tag -s -m 'v0.8.26' v0.8.26; git push --tags`
```

Comment:
I would advise against using `git push --tags` (you don't want to know the extra tags I have on my machine :)).
It would be safer to specify the tag and remote explicitely and using a VAR prevents typos.

Untested (can be collapsed to a single line):
```
VERSION=0.8.26
REMOTE=origin
git tag -s -m "v${VERSION}" v${VERSION}; git push "v${VERSION}" ${REMOTE}
```

It may be a good option to bring back the `release.sh` with the content above.
We can use `REMOTE=origin` as default and complain if version is not set.

In the script, I would suggest to:
- print a summary of what will be done (so we can stop typical errors such as being about to create tag `vv1.2.3` instead of `v1.2.3` because we passed the wrong `VERSION`
- pause with a read and wait for confirmation to move on. Could be done at start or just before the push
- apply